frontend/src/AddTagView.svelte

```diff
@@ -0,0 +1,222 @@
+<style>
+  button {
+    padding-top: 0.5em;
+    padding-bottom: 0.5em;
+    padding-left: 1em;
+    padding-right: 1em;
+  }
+
+  button:hover,
+  button:focus {
+    outline: none;
+    box-shadow: inset 1px 1px 0 var(--main-fg-color),
+      inset -1px -1px 0 var(--main-fg-color);
+  }
+
+  button:active {
+    box-shadow: inset 2px 2px 0 var(--main-fg-color),
+      inset -2px -2px 0 var(--main-fg-color);
+  }
+
+  .container {
+    min-height: 100%;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: nowrap;
+    justify-content: center;
+    align-items: stretch;
+    align-content: center;
+  }
+
+  .inputs {
+    flex-grow: 1;
+  }
+
+  .inputs *:first-child {
+    margin-top: 0;
+  }
+
+  .actions {
+    margin-top: 2em;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: space-evenly;
+    align-items: center;
+    align-content: center;
+  }
+
+  label {
+    margin-bottom: 1em;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: space-evenly;
+    align-items: baseline;
+    align-content: center;
+    white-space: nowrap;
+  }
+
+  .nowrap {
+    white-space: nowrap;
+  }
+
+  .error {
+    margin-top: 2em;
+  }
+
+  h1 {
```

Comment:
There are a bunch of unused CSS selectors that cause warnings during the `npm run build`, and that should be removed.

```
(!) Plugin svelte: Unused CSS selector "label"
src/AddTagView.svelte
47:   }
48:
49:   label {
      ^
50:     margin-bottom: 1em;
51:     display: flex;
(!) Plugin svelte: Unused CSS selector ".nowrap"
src/AddTagView.svelte
58:   }
59:
60:   .nowrap {
      ^
61:     white-space: nowrap;
62:   }
(!) Plugin svelte: Unused CSS selector "h1"
src/AddTagView.svelte
66:   }
67:
68:   h1 {
      ^
69:     text-transform: uppercase;
70:     font-weight: bold;
(!) Plugin svelte: Unused CSS selector ".center"
src/AddTagView.svelte
81:   }
82:
83:   .center {
      ^
84:     display: flex;
85:     flex-direction: column;
(!) Plugin svelte: Unused CSS selector "input[type="file"]"
src/AddTagView.svelte
114:   }
115:
116:   input[type="file"] {
       ^
117:     display: none;
118:   }
created public/build/bundle.js in 7.2s
```